### ISSUE TITLE
[RFC] If --query & --select-1 are specified and peco shows exactly 1 line, select the line

### ISSUE THREAD

**Issue by nitoyon** [peco/peco#281]
When `--query` & `--select-1` options is specified and the result contains exactly 1 line, it would be useful if peco selects the only line.
## Example:

pcd.bat = dir + cd on Windows

``` bat
@echo off

if "%1" neq "" (
  for /f %%i in ('dir /B /AD ^| peco --select-1 --query %1') do (
    cd %%i
    break
  )
) else (
  for /f %%i in ('dir /B /AD ^| peco') do (
    cd %%i
    break
  )
)
```

(ex) execute `pcd` without arguments.

```
C:\> pcd
QUERY> 
Program Files
Program Files (x86)
ProgramData
Users
Windows
```

(ex) execute `pcd 86` [BEFORE]

```
C:\> pcd 86
QUERY> 86
Program Files (x86)
```

(ex) execute `pcd 86` [AFTER]

```
C:\> pcd 86
C:\Program Files (x86)>
```

**Comment by lestrrat** [comment]
In theory this is understandable, but if we do this, we would have to treat the first run query differently than all others, and I think it's not that trivial to implement.

**Comment by lestrrat** [comment]
I may have inadvertently fixed this in the latest merge #304

**Comment by kaushalmodi** [comment]
From what I understand, this is not supported, correct?

I would like `--select-1` to work in a case like this:

```
echo "a\nb" | peco --select-1 --query=a
```

Currently above still shows the `peco` interface even where the filtered results will have only 1 row.

But in below case, where peco has only one row in input, `--select-1` does its job. *I am curious what use case this would be..*.

```
echo "a" | peco --select-1
```
